### ISSUE TITLE
I.L. FE: Add real content to app

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#13750]
**STATUS 6/30: awaiting stakeholder approval of final content**

## Description

Adding content to app is blocked by: 
1. Final content
2. Stakeholder review

### User story

**AS A** PO/PM
**I WANT** the final content to be added to the app
**SO THAT** the experience is optimal for Veterans.

### Notes / background

**Content source of truth**
Stakeholders reviewed content in Word. This [Word doc](https://dvagov-my.sharepoint.com/:w:/g/personal/wesley_rowe_va_gov/EVGex-FjRcBDo11XO6EV6zwBbH8yKyLCfPYVmwBhBGex9Q?e=EUNd2O) (CAG required) is the draft they approved. One comment on the Review Info screen is being discussed, ignore it for now.

CAIA (Sitewide Content) [intake ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/51132).

## Acceptance criteria

- [x] Each screen in both current-year and previous-year flows matches exactly what's in the Word doc linked above.

**Comment by wesrowe** [comment]
Waiting for final content to gel. [Slack thread](https://dsva.slack.com/archives/C52CL1PKQ/p1685475663652649) with Danielle has a couple outstanding items as of 6/2.

**Comment by jilladams** [comment]
From planning, @davidconlon to follow up with Danielle re: status on content / timing.

**Comment by wesrowe** [comment]
Update 7/3: Stakeholders proposed minor changes to content about dependents; Danielle to provide (and unblock this ticket).

**Comment by randimays** [comment]
@wesrowe A minor technical snafu with the Word doc. I can't copy/paste text from the Windows CAG instance over to my Mac, where the code lives. I downloaded a copy and emailed it to myself, though. If there are any changes past this point, could we please ask the authors to highlight / otherwise call out where they are made?

**Comment by randimays** [comment]
@wesrowe The `Learn more about income limits and deductions` link on the Results page links here: `preview-prod.vfs.va.gov/preview?nodeId=57286`. Just dropping a note that we'll need to get this updated before launch.

**Comment by wesrowe** [comment]
@randimays, thanks, I just created a "Launch IL" [ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14294) and added fixing this link as an AC. Please add any launch-related thoughts you have to that ticket going forward!

**Comment by randimays** [comment]
We need some fine-tuning on the verbiage for the past flow specifically on the Results page (and maybe another spot or two?). [See this Slack thread](https://dsva.slack.com/archives/C52CL1PKQ/p1688760490696529) for discussion and more context. Going to put this ticket on hold until we can get the content finalized.

**Comment by wesrowe** [comment]
_(comment that was threaded on the wrong ticket)_

@randimays, thanks for reading this with a critical eye. Unless we have another version somewhere, I think 

Looking at the full text of the current-year results screen, I think we might want to tweak it a bit more for past-years.

> **Your income limits for [selected year]**
>
>We use your last year’s household income, minus certain deductions, to determine your VA health care eligibility and costs. Select your income range to find out how your income may affect your eligibility and costs. But remember, the only way to know for sure is to apply.  
>
> How to estimate your household income range: 
> 
> - Add up all the money your household earned in [previous year] from jobs and other sources. Your household includes you and your spouse and dependents if you have them. 
> - Then subtract any deductions you had in [previous year]. You can deduct some health care costs you didn’t get paid back for, expenses for your own college or vocational training, and any funeral or burial expenses for a spouse or dependent child. 

Things I noticed:
- I agree with Randi's observation that we should adjust "last year's household income" to something like "previous year's"
- It doesn't make sense to me to urge a past-year viewer to apply. We think they already have healthcare to even be interested in past years
- present tense feels weird In "Your household includes you and your spouse and dependents if you have them." It's in the past, it should be clearer that their dependents that year is what matters. 

@thejordanwood, do you remember seeing any versions of the previous-year copy docs that had adjusted the language for the past year flow? If not, we seem to need to do a cycle with Danielle.

**Comment by wesrowe** [comment]
@DanielleThierryUSDSVA, we have discovered that we probably need a little more content adjustment for the past-year flow of Income Limits. (Comment above.) It's not many words that need to be adjusted.

(Please keep @thejordanwood and @randimays in the loop on this, as I'll be out on PTO for the 2nd half of the sprint.)

**Comment by jilladams** [comment]
> Looks like this came in during last week when we were down multiple people on CAIA so I just haven't gotten to it yet. You can always feel free to reach out in slack if you don't hear from me. This is a pretty easy fix so I'll try to respond in the ticket tomorrow. Thanks!
https://dsva.slack.com/archives/C01K37HRUAH/p1689626341533289?thread_ts=1689623107.722339&cid=C01K37HRUAH

**Comment by DanielleThierryUSDSVA** [comment]
Hi there! I've added a Results for past years screen in the [main content doc](https://dvagov-my.sharepoint.com/:w:/g/personal/david_conlon_va_gov/Ebc_XmJgMe9Fu2cumsd6kHwBv6mqAesmwr0ZzWPyfJVMuw?e=jZKCB2) and tracked/highlighted changes so it's easy to spot them. 

In the "How to estimate your household income range:" bullets and in the "Select your household income range" header, I think ideally we'd want to have the actual previous year listed. So, if the past year is 2022, we'd use 2021 in those places. Thought that might introduce too much complexity, so I've just used "previous year" in those spots. If you'd like to try adding the year instead there, let me know.

**Comment by randimays** [comment]
@DanielleThierryUSDSVA Thank you for the updated content! There are a couple more spots that I needed to check in about. The Zip code screen and the Dependents screen have a couple of lines that may need adjustment for the past year flow.

> What was your zip code last year?

> How many dependents did you have last year?

Instead of "last year," should it say "in [selected year - 1]"?

**Comment by DanielleThierryUSDSVA** [comment]
@randimays Agree! Doc is updated with that approach in the 2 question screens + the results screen (new edits highlighted in blue for ease of spotting). Thanks!

**Comment by randimays** [comment]
PR is up for review. Going to get everything merged to staging and have everyone review unimpeded by the API not working in review instances.

**Comment by wesrowe** [comment]
I verified that the copy on Staging is exactly what was expected based on these sources:
- current-year flow from [here](https://dvagov-my.sharepoint.com/:w:/r/personal/wesley_rowe_va_gov/Documents/Documents/Income%20Limits/Income%20limits%20wizard%20content%20-%20post-Stakeholders.docx?d=we1c79e51456343c0a35d573ba115eb3c&csf=1&web=1&e=y0lRRc)

Closing ticket!
- past-year flow from [here](https://dvagov-my.sharepoint.com/:w:/r/personal/david_conlon_va_gov/Documents/Income%20Limits/Income%20limits%20wizard%20content.docx?d=w625e3fb7316045efbb672e9ac77a907c&csf=1&web=1&e=ExOHNq)

**Comment by wesrowe** [comment]
Reopening ticket – I missed a slack response that we want to update links to stay in the current environment.

**Comment by randimays** [comment]
Links are relative now in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/24972) but it seems like the Github runners are down. The pipeline is stalled right now trying to get through the CI steps. Will be merging this ASAP.
